cmd/Makefile.am

```diff
@@ -60,7 +64,8 @@ fmt:: $(foreach dir,$(subdirs),$(wildcard $(srcdir)/$(dir)/*.[ch]))
 # installing a fresh copy of snap confine and the appropriate apparmor profile.
 .PHONY: hack
 hack: snap-confine/snap-confine-debug snap-confine/snap-confine.apparmor snap-update-ns/snap-update-ns snap-seccomp/snap-seccomp snap-discard-ns/snap-discard-ns snap-device-helper/snap-device-helper snapd-apparmor/snapd-apparmor
-	sudo install -D -m 4755 snap-confine/snap-confine-debug $(DESTDIR)$(libexecdir)/snap-confine
+	sudo install -D -m 755 snap-confine/snap-confine-debug $(DESTDIR)$(libexecdir)/snap-confine
+	sudo setcap "$(SNAP_CONFINE_CAPS)" $(DESTDIR)$(libexecdir)/snap-confine
```

Comment:
I wonder if we could get rid of `make hack`. Or make a shell script that does `make install DESTDIR=/some/temporary`, then copies the files from there.

Reply:
hack also reloads the apparmor profile if neded, in any case, a followup PR rather than this one